src/store/sagas/widgetConfig.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 Cognite AS
+import { getByUser, update } from 'components/widgetCRUD/repo/widgetConfLSRepo';
+import { select, put, Effect } from 'redux-saga/effects';
+import { RootState } from 'StoreTypes';
+import { setWidgetConfigs, setAlerts } from 'store/actions/app';
+import get from 'lodash/get';
+import { MESSAGES } from 'constants/messages';
+
+const getUserId = (state: RootState) => state.authState.userInfo?.name;
+
+export function* showAlert(msg: string) {
+  yield put(
+    setAlerts({
+      type: 'error',
+      text: msg,
+      hideApp: false,
+    })
+  );
+}
+
+/**
+ * Retriew widget configs by userId and assetId
+ */
+export function* machineIdChanged({ payload }: Effect) {
```

Comment:
why cant this be called load loadWidgetConfigs rather than machineIdChanged? 
This is more consistent, and more suitable in Saga's context. Please name functions, vars, classes, etc in way that makes sense to the current context.